controllers/managedocs_controller.go

```diff
@@ -121,6 +121,7 @@ type ManagedOCSReconciler struct {
 	k8sMetricsServiceMonitorAuthSecret *corev1.Secret
 	namespace                          string
 	reconcileStrategy                  v1.ReconcileStrategy
+	CustomerNotificationTemplate       string
```

Comment:
```suggestion
	CustomerNotificationTemplatePath       string
```

also can we move this declaration next to `AlertSMTPFrom`

Reply:
Done

Reply:
This change seems not scalable, instead of having the entire path in the `ManagedOCSReconciler` can we have just the template path ?
something like
TemplatePath  = "templates/"

and read the necessary file by concatenating under the appropriate method. In this case, we can avoid declaring multiple  variables under the `ManagedOCSReconciler` when template file grows